cloud-optimized-geotiffs/accessing-cogs-in-r-terra.ipynb

```diff
@@ -0,0 +1,526 @@
+{
```

Comment:
<p>Check if specifying driver makes it faster (this is the case with geopandas and sf)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Looks like in the case of <code>geopandas</code>, the default is <code>fiona</code> backing vs <code>pyogrio</code> .  In the case of <code>terra</code>, it appears as though the GDAL wrapping is handled directly in terra source (<a href="https://github.com/rspatial/terra/blob/master/src/gdalio.cpp" target="_blank">https://github.com/rspatial/terra/blob/master/src/gdalio.cpp</a>) and so probably doesn't admit of any speedup of that sort. I also couldn't find anything in the documentation to suggest it, so I'm thinking there's no speed up to be had in this fashion</p>

Reply:
? pyogrio should be default in recent geopandas - but I'm not talking about fiona vs pyogrio. I'm talking about the step where GDAL guesses the format. If you pass a driver it doesn't have to guess.

Reply:
this is minor and we can look into it more later